app/assets/stylesheets/members/index.css.scss

```diff
@@ -53,6 +95,19 @@ $parallax-background-height: $parallax-window-height;
   width: 100%;
 }
 
+@include breakpoint($mobile) {
+  .parallax-background {
+    background: url("parallax.jpg");
+    background-size: cover;
+    background-color: beige;
```

Comment:
Color `beige` should be written in hexadecimal form as `#f5f5dc`<br>Color literals like `beige` should only be used in variable declarations; they should be referred to via variable everywhere else.